lib/node_modules/@stdlib/stats/base/dsnanmean/benchmark/benchmark.js

```diff
@@ -39,17 +45,7 @@ var dsnanmean = require( './../lib/dsnanmean.js' );
 * @returns {Function} benchmark function
 */
 function createBenchmark( len ) {
-	var x;
-	var i;
-
-	x = new Float32Array( len );
-	for ( i = 0; i < x.length; i++ ) {
-		if ( randu() < 0.2 ) {
-			x[ i ] = NaN;
-		} else {
-			x[ i ] = ( randu()*20.0 ) - 10.0;
-		}
-	}
+	var x = uniform( len, -10.0, 10.0, options );
```

Comment:
This doesn't match the original random value generation. Need to use `filledarrayBy` and a custom random function.